### PR TITLE
ST-782 Summary page & respondent

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ST-782

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/NotifyParties.java [hunk 1]
```diff
@@ -49,8 +48,8 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(CaseDetails<CaseDa
     private boolean checkNull(CaseData data) {
         return null != data.getCicCase()
             && (null == data.getCicCase().getNotifyPartySubject() || data.getCicCase().getNotifyPartySubject().isEmpty())
-            && (null == data.getCicCase().getNotifyPartyApplicant() || data.getCicCase().getNotifyPartyApplicant().isEmpty())
-            && (null == data.getCicCase().getNotifyPartyRepresentative() || data.getCicCase().getNotifyPartyRepresentative().isEmpty());
+            && (null == data.getCicCase().getNotifyPartyRepresentative() || data.getCicCase().getNotifyPartyRepresentative().isEmpty())
+            && (null == data.getCicCase().getNotifyPartyRespondent() || data.getCicCase().getNotifyPartyRespondent().isEmpty());
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/NotifyParties.java:52]: Use the apache StringUtils.isNotBlank (null, "" or whitespace) or isNotEmpty (null, ""), perhaps? Would make it more readable, too.

**Review by osheaf01 — CHANGES_REQUESTED** [review]
Think you should use StringUtils helper method?
Otherwise LGTM

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/NotifyParties.java (hunk 1 above)
Comment (ElifDenizEsea) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/NotifyParties.java:52]: they are not strings, they are enum sets

Comment (osheaf01) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/NotifyParties.java:52]: Ah, OK, then I suggest you use CollectionUtils.isEmpty, which is null-safe.

Replace 

`&& (null == data.getCicCase().getNotifyPartyApplicant() || data.getCicCase().getNotifyPartyApplicant().isEmpty())`

With

`&& (CollectionUtils.isEmpty(data.getCicCase().getNotifyPartyApplicant()))`

**Review by osheaf01 — APPROVED** [review]
LGTM

**Review by sjamihmcts — APPROVED** [review]

**Pull request merged**